fud/fud/config.py

```diff
@@ -57,6 +57,26 @@
             "ssh_username": "",
             "remote": None,
         },
+        "xclbin": {
+            "file_extensions": [".xclbin"],
+            "mode": "hw_emu",
+            "device": "xilinx_u50_gen3x16_xdma_201920_3",
+            "temp_location": "/tmp",
+            "ssh_host": "",
+            "ssh_username": "",
+            "save_temps": None,
+        },
+        "wdb": {
+            "file_extensions": [".wdb"],
+            "mode": "hw_emu",
+            "ssh_host": "",
+            "ssh_username": "",
+            "host": None,
+            "save_temps": None,
+            "xilinx_location": "/scratch/opt/Xilinx/2020.2",
```

Comment:
Is this specific for gorgonzola?

Reply:
for havarti. I can remove this default, but the logic was that it's overridable + it's the sane default for most people that will be using `fud` in the short term future. For future-proofing though the right thing is to not set this